packages/ocaml-config/ocaml-config.2/opam

```diff
@@ -12,10 +12,12 @@ bug-reports: "https://github.com/ocaml/opam/issues"
 depends: [
   "ocaml-base-compiler" {>= "4.12.0~"} |
   "ocaml-variants" {>= "4.12.0~"} |
-  "ocaml-system" {>= "4.12.0~"}
+  "ocaml-system" {>= "4.12.0~"} |
+  "dkml-base-compiler" {>= "4.12.0~"}
 ]
-substs: "gen_ocaml_config.ml"
+substs: ["gen_ocaml_config.ml" "gen_dkml_config.ml"]
 extra-files: [
   ["gen_ocaml_config.ml.in" "md5=a4b41e3236593d8271295b84b0969172"]
-  ["ocaml-config.install" "md5=8e50c5e2517d3463b3aad649748cafd7"]
+  ["gen_dkml_config.ml.in" "sha256=dbd74b9cf2a16fa967c02f038801f5b563dcccad109799adfdf7f6dc60c574fa"]
```

Comment:
All of the changes here can be welcomed in `gen_ocaml_config.ml.in`, there's no need for a separate script 🙂

Reply:
Yep!